lambdas/db-migration/src/index.ts

```diff
@@ -16,30 +18,54 @@ const getRequiredEnvVar = (name: string, env: NodeJS.ProcessEnv): string => {
   throw new Error(`The ${name} environment variable must be set`);
 };
 
-const getConnectionConfig = (env: NodeJS.ProcessEnv): Knex.PgConnectionConfig => ({
-  host: getRequiredEnvVar('PG_HOST', env),
-  user: getRequiredEnvVar('PG_USER', env),
-  // TODO Get this value from secrets manager
-  password: getRequiredEnvVar('PG_PASSWORD', env),
-  database: getRequiredEnvVar('PG_DATABASE', env),
-});
+const getSecretConnectionConfig = async (SecretId: string): Promise<Knex.PgConnectionConfig> => {
+  const secretsManager = new AWS.SecretsManager();
+  const response = await secretsManager.getSecretValue(
+    { SecretId } as AWS.SecretsManager.GetSecretValueRequest
+  ).promise();
+  if (response.SecretString === undefined) {
+    throw new Error('Database credentials are undefined!');
+  }
+  const dbAccessMeta = JSON.parse(response.SecretString);
+  return {
+    host: dbAccessMeta.host,
+    user: dbAccessMeta.username,
+    password: dbAccessMeta.password,
+    database: dbAccessMeta.database,
+  };
+};
+
+const getConnectionConfig = async (env: NodeJS.ProcessEnv): Promise<Knex.PgConnectionConfig> => {
```

Comment:
@marc mentioned that he didn't want to move this helper to a package, but I foresee us using this exact invocation across multiple Lambdas, including data-migration. I could see it moving to the `db` package but it's not essential

Reply:
Hmm, I think moving it to the DB package is fine, so long as we leave connection timeouts and such as a param-configurable thing.    There's going to be configuration use cases for Core DB access and these scripts depending on the deployment setup.